research/flamby/fed_isic2019/fedavg/server.py

```diff
@@ -36,6 +37,28 @@ def main(config: Dict[str, Any], server_address: str, checkpoint_stub: str, run_
     client_manager = SimpleClientManager()
     model = Baseline()
 
+    log(INFO, f"if pretrain: {pretrain}")
+    if pretrain:
+        dir = (
+            "/ssd003/projects/aieng/public/FL_env/models/fed_isic2019/fedavg/hp_sweep_results/lr_0.001/"
+            + run_name
+            + "/server_best_model.pkl"
+        )
+        fedavg_model_state = torch.load(dir).state_dict()
```

Comment:
I would suggest encapsulating this load into a method.

Reply:
I will delete this from this pr and put it up on separate pr. This was not supposed to be here sorry.